Documentation/_static/helm-reference.css

```diff
@@ -0,0 +1,12 @@
+#helm-reference .wy-table-responsive table td {
+    white-space: normal!important;
+    word-wrap: break-word!important;
+}
+
+#helm-reference .wy-table-responsive table td:nth-child(1) {
+    max-width: 240px!important;
+}
+
+#helm-reference .wy-table-responsive table td:nth-child(2) {
+    max-width: 320px!important;
+}
```

Comment:
The result (at least in my browser) was not very ergonomic because the default values were hidden.

Adding the following, improves it for me:
```
#helm-reference table td .pre {
    white-space: pre-wrap;
}

#helm-reference table {
    width: 100%;
    table-layout: fixed;
}
```

However, do we really need a table? IMO I think the end result would be much more readable if we had the various fields as sections instead of table cells.

Reply:
This looks much better indeed — I've amended the commit. As for the table, I agree with you but a table is what we get from https://github.com/norwoodj/helm-docs — maybe we can turn that into sections with a little work, but also maybe as part of a separate PR?

Reply:
Ah right, because the file is auto-generated. Yeah, probably best to do it in a different PR and go with what we have for now.